src/libs/actions/Report.js

```diff
@@ -368,6 +375,14 @@ function fetchChatReportsByIDs(chatList) {
             PersonalDetails.getFromReportParticipants(Object.values(simplifiedReports));
 
             return _.map(fetchedReports, report => report.reportID);
+        })
+        .catch((err) => {
+            if (err.message === 'inacessible' && shouldRedirectIfInacessible) {
+                Log.info('[Report] Report data is inacessible.', true);
```

Comment:
Let's maybe get rid of this log since it will actually print to the server and we don't need it to do that in this case.